docs/internal-developers/automatations/dependabot.md

```diff
@@ -0,0 +1,10 @@
+
+# Dependabot
+
+This repository uses [`Dependabot`](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates) to help with keeping all the package dependencies (NPM, Composer, GitHub Actions) up to date. Without this in place, it's very easy to let the package versions we're using go stale and creates a backlog of chores for updating those in the future. It is essential to keep dependencies updated to avoid security problems and lower overall upgrade costs.
+The process is automated: Dependabot creates a branch and a PR with a package bump in package.json. A new package-lock.json is created. Automated tests are executed. Also, Dependabot will create a maximum of 10 PRs for each ecosystem (NPM, Composer, GitHub Actions).
+
+It is the responsibility of the porter reviews these PRs weekly and merges/rejects them.
```

Comment:
```suggestion
It is the responsibility of the porter to review these PRs weekly and merge/reject them.
```

Should we really make mention of the porter role here since this is the public repo and I don't see any other mention anywhere? If so, should we link to our team handbook? CC @Aljullu

Reply:
Mmm, good point! Let's wait for @Aljullu feedback!

Reply:
We do have some mentions to the porter in the release checklist. I think it's fine. I wouldn't link to the Team Handbook, though.